### PR TITLE
Follow up on default LoadBalancer Algorithm

### PR DESCRIPTION
**Pull request by MustafaSaber**
follow up on https://github.com/zalando/skipper/pull/2413

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -450,7 +450,7 @@ func NewConfig() *Config {
 	flag.StringVar(&cfg.KubernetesRedisServiceName, "kubernetes-redis-service-name", "", "Sets name for redis to be used to lookup endpoints")
 	flag.IntVar(&cfg.KubernetesRedisServicePort, "kubernetes-redis-service-port", 6379, "Sets the port for redis to be used to lookup endpoints")
 	flag.StringVar(&cfg.KubernetesBackendTrafficAlgorithmString, "kubernetes-backend-traffic-algorithm", kubernetes.TrafficPredicateAlgorithm.String(), "sets the algorithm to be used for traffic splitting between backends: traffic-predicate or traffic-segment-predicate")
-	flag.StringVar(&cfg.KubernetesDefaultLoadBalancerAlgorithm, "kubernetes-default-lb-algorithm", "roundRobin", "sets the default loadbalancer algorithm to be used for traffic splitting between backends: roundrobin, random, consistentHash, powerOfRandomNChoices. default: roundRobin")
+	flag.StringVar(&cfg.KubernetesDefaultLoadBalancerAlgorithm, "kubernetes-default-lb-algorithm", kubernetes.DefaultLoadBalancerAlgorithm, "sets the default loadbalancer algorithm to be used for traffic splitting between backends: roundrobin, random, consistentHash, powerOfRandomNChoices.")
```
Comment (AlexanderYastrebov) [inline config/config.go:453]: roundrobin -> roundRobin

Comment (AlexanderYastrebov) [reply config/config.go:453]: I think description is copy-paste from KubernetesBackendTrafficAlgorithmString - the algorithm distributes requests between _endpoints_.
Lest make it `sets the default algorithm to be used for load balancing between backend endpoints`

Comment (AlexanderYastrebov) [reply config/config.go:453]: Please also fix Options.DefaultLoadBalancerAlgorithm godoc
https://github.com/zalando/skipper/blob/0d5031e3c382f29d51ab43621a57bb0a9e431bff/skipper.go#L277-L278

Comment (AlexanderYastrebov) [reply config/config.go:453]: and https://github.com/zalando/skipper/blob/0d5031e3c382f29d51ab43621a57bb0a9e431bff/dataclients/kubernetes/kube.go#L223-L224

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by MustafaSaber** [comment]
Will merge once PR finishes successfully

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**